### PR TITLE
fix trusted status indication

### PR DESCRIPTION
**Pull request by adigaboy**
fixed trusted check to be similar to:
https://github.com/jupyterlab/jupyterlab/blob/19fa54fcbf3e684d7c4466807af549ba0d565ad2/packages/notebook/src/truststatus.tsx#L240-L259

includes check cell type if code
some total code cells
check and sum trusted cells

### CODE REVIEW

File: packages/notebook-extension/src/trusted.tsx [hunk 1]
```diff
@@ -17,16 +17,19 @@ const isTrusted = (notebook: Notebook): boolean => {
...
+  for (const current_cell of cells) {
+    if (current_cell.type !== 'code') {
+      continue;
     }
-  }, 0);
+    total++;
+    if (current_cell.trusted) {
+      trusted++;
+    }
+  };
```
Comment (krassowski) [inline packages/notebook-extension/src/trusted.tsx:31]:
```suggestion
  }
```

File: packages/notebook-extension/src/trusted.tsx [hunk 2]
```diff
@@ -17,16 +17,19 @@ const isTrusted = (notebook: Notebook): boolean => {
...
   const cells = Array.from(model.cells);
+  let total = 0;
+  let trusted = 0;
 
-  const trusted = cells.reduce((accum, current) => {
-    if (current.trusted) {
-      return accum + 1;
-    } else {
-      return accum;
+  for (const current_cell of cells) {
```
Comment (krassowski) [inline packages/notebook-extension/src/trusted.tsx:23]: It would be good to follow the `camelCase` convention here, this is `currentCell`

**Review by krassowski — COMMENTED** [review]
Thanks @adigaboy. I left some minor comments on code style :)

**Comment by jtpio** [comment]
bot please update playwright snapshots

**Comment by jtpio** [comment]
Thanks!

The remaining test failure is not related and is being addressed in https://github.com/jupyter/notebook/pull/7037.

**Comment by adigaboy** [comment]
My pleasure!

Will look for other issues to fix also :)

**Pull request merged**